presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -431,7 +431,7 @@ bool SystemConfig::logZombieTaskInfo() const {
 }
 
 uint32_t SystemConfig::logNumZombieTasks() const {
-  return optionalProperty<uint32_t>(kLogZombieTaskInfo).value();
+  return optionalProperty<uint32_t>(kLogNumZombieTasks).value();
```

Comment:
I run into this issue and fix in https://github.com/prestodb/presto/pull/19864. Thanks!